tests/callbacks/callback_settings.py

```diff
@@ -223,5 +224,8 @@ def get_cb_model_and_datasets(cb: Callback,
             )
         return (configure_tiny_gpt2_hf_model(), dummy_gpt_lm_dataloader(size=dl_size),
                 dummy_gpt_lm_dataloader(size=dl_size))
-    return (SimpleModel(), DataLoader(RandomClassificationDataset(size=dl_size), **default_dl_kwargs),
+    model = SimpleModel()
+    if isinstance(cb, FreeOutputs):
+        model.get_metrics = lambda is_train=False: {}
```

Comment:
```suggestion
        model.get_metrics = lambda x: {}
``` 
Can it just be this?

Reply:
Needs a default value as there are places in the codebase where `is_train` is not specified. Plus, just nice to match signature